src/System.Private.CoreLib/shared/System/Runtime/InteropServices/MemoryMarshal.cs

```diff
@@ -86,12 +87,12 @@ public static bool TryGetArray<T>(ReadOnlyMemory<T> memory, out ArraySegment<T>
         /// <param name="memory">The memory to get the manager for.</param>
         /// <param name="manager">The returned manager of the <see cref="ReadOnlyMemory{T}"/>.</param>
         /// <returns>A <see cref="bool"/> indicating if it was successful.</returns>
-        public static bool TryGetMemoryManager<T, TManager>(ReadOnlyMemory<T> memory, out TManager manager)
+        public static bool TryGetMemoryManager<T, TManager>(ReadOnlyMemory<T> memory, out TManager? manager) // TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/26761
             where TManager : MemoryManager<T>
         {
-            TManager localManager; // Use register for null comparison rather than byref
+            TManager? localManager; // Use register for null comparison rather than byref
```

Comment:
@ahsonkhan, the previous code here didn't make sense to me... the local wasn't actually being used.  Is there a reason it was correct as it was, or is my change ok?  Same below.

Reply:
Your change is correct and was the initial expectation (see https://github.com/dotnet/coreclr/pull/16455/files#diff-85d9d06bed20dc229c87dc454ec35fe4R61)

I missed this change during the bulk parameter renaming/mirroring between corefx/coreclr:
https://github.com/dotnet/corefx/pull/28160/files#diff-28901840fe6c71c0ba1c56e2cf99bfdaR64

Commit to coreclr with the mistake: https://github.com/dotnet/coreclr/commit/4113c35a3fcfb3952da7e088158a99f003cf9cd8

Good catch!